### PR TITLE
Release 4.1.0

### PR DESCRIPTION
**Pull request by Blackbaud-TrevorBurch**

### CODE REVIEW

**Comment by blackbaud-ado** [comment]
Preview build at [https://developer.blackbaud.com/skyux-split-view?_sp.spa=pr-release-4.1.0](https://developer.blackbaud.com/skyux-split-view?_sp.spa=pr-release-4.1.0)

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,8 @@
+# 4.1.0 (2020-09-11)
+
+- Added the `bindHeightToWindow` input property to the split view component to allow for split views on the main application page to be bound to the window's height. [#37](https://github.com/blackbaud/skyux-split-view/pull/37)
```
Comment (Blackbaud-DeniseCisneros) [inline CHANGELOG.md]: Does this change meaning?

Added the `bindHeightToWindow` input property to the split view component to bind split views on the main application page to the window's height.

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,3 +1,8 @@
...
+- Fixed the split view component to set responsive styles after the split view has fully rendered. [#43](https://github.com/blackbaud/skyux-split-view/pull/43)
```
Comment (Blackbaud-DeniseCisneros) [inline CHANGELOG.md]: Minor suggestion. Delete "has" and change "rendered" to "renders."

**Review by Blackbaud-DeniseCisneros — COMMENTED** [review]
A couple suggestions

**Review by Blackbaud-DeniseCisneros — APPROVED** [review]

**Comment by blackbaud-ado** [comment]
Preview build at [https://developer.blackbaud.com/skyux-split-view?_sp.spa=pr-release-4.1.0](https://developer.blackbaud.com/skyux-split-view?_sp.spa=pr-release-4.1.0)

**Pull request merged**